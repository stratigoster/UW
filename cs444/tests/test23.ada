PACKAGE foo IS
  TYPE data IS RECORD
    a : integer;
    b : boolean;
  END RECORD;
  d : data;
  i : integer := d.a;
  j : integer := d.b;
--               ^^^
-- incompatible assignment of boolean to integer

  k : integer := d.blah;
--               ^^^^^^
-- no field `blah' in record data
END foo;
