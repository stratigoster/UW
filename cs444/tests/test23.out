***ERROR***:8:5:	Incompatible types in assignment of ` universal boolean ' to ` universal integer '
***ERROR***:12:25:	Record ` data ' doesn't contain field ` blah '

------
Compilation error
------
